### PR TITLE
feat: Fixes #308. Change amount font size on Send page to fit in input field

### LINKED ISSUE #308: Change font size of amount to be send when over 9 digits are entered
I have an account with THIBCoin, and I've been asked to transfer all of it to another account to cover the cost of a ransom or face consequences.
In the UI I go to the "Send THIBCoin" page where it shows I only have a balance of 1,123.460,932.
I know this transaction has to be precise, but luckily I'm a long-term user so I know the values shown on this page in UI are rounded to 6 decimal places, so to make sure I enter the correct value I click the link provided by https://github.com/paritytech/fether/issues/98 (instead of having to use a search engine or bookmark to find the etherscan link where I get can the exact account balance that has been demanded (note that the value shown on Etherscan is the base units https://kovan.etherscan.io/token/0x4a6e6c3868a279e1d9047b42c3fb356ff4680003?a=0x005072Fb985cc64d8A1Ce00ECe26279f73c12F8a.
So I enter the exact account balance, which is actually 1,123.460,932,110 THIB.

screen shot 2018-12-19 at 6 17 22 pm

But it clips the value as shown above.

So propose to reduce the font-size when the value entered is >9 digits long. This involves changing 
the CSS `input.form_field_amount[type='number'] from its existing font-size: 2.98598rem;

If we reduce it to font size of 2rem it fits 14 decimal places, or if we reduce it to 1.4rem it fits the smallest unit.

screen shot 2018-12-19 at 6 23 21 pm

Alternative is to permanently reduce the font size to some value.

### PR DESCRIPTION
**Pull request by ltfschoen**
* Switches to font size that fits a value as small as 1 Wei on the screen when more than 9 digits entered

* Switches back to default font size when amount of digits entered is 9 or less again

* Caters for when user unchecks the Max Value button

### CODE REVIEW

File: packages/fether-react/src/Send/TxForm/TxForm.js [hunk 1]
```diff
@@ -61,6 +65,23 @@ class Send extends Component {
     }
   });
 
+  changeAmountFontSize = () => {
+    const amountElement = document.getElementById('amount');
```
Comment (amaurymartiny) [inline packages/fether-react/src/Send/TxForm/TxForm.js]: No! React uses its own virtual DOM, so we never directly modify the DOM.

Comment (ltfschoen) [reply packages/fether-react/src/Send/TxForm/TxForm.js]: I've been looking into this but had difficulties using refs. I've also tried controlled components, and mutators unsuccessfully so far

Comment (amaurymartiny) [reply packages/fether-react/src/Send/TxForm/TxForm.js]: An idea: on `<Field name="amount">`, in its className field, you can add a CSS class when the `values.amount`'s length is longer than some value.

**Comment by ltfschoen** [comment]
@amaurymartiny I've pushed a commit that addresses your review comment

**Comment by ltfschoen** [comment]
I've merged latest from master

File: packages/fether-react/src/Send/TxForm/TxForm.js [hunk 2]
```diff
@@ -42,6 +44,13 @@ class Send extends Component {
     showDetails: false
   };
 
+  handleSubmit = values => {
```
Comment (amaurymartiny) [inline packages/fether-react/src/Send/TxForm/TxForm.js]: Revert. handleSubmit is already somewhere else in the class

**Pull request merged**